code/modules/projectiles/gun.dm

```diff
@@ -114,19 +117,26 @@
 /obj/item/gun/attackby(obj/item/I, mob/living/user, params)
 	if(!istool(I) || user.a_intent != I_HURT)
 		return FALSE
+	if(I.get_tool_quality(QUALITY_HAMMERING) && !serial_scribbled)
```

Comment:
```suggestion
	if(I.get_tool_quality(QUALITY_HAMMERING) && serial_type)
```

Reply:
?

Reply:
I do need it , i dont want people getting shot over some gun having no serial and being catalogued as "scribbled"

Reply:
If you want to differentiate between guns that have serial removed, and those that never had it to begin with, just `null` that value in this proc and check for `else if(isnull(serial_type))` on `examine()`.
There is like 9000 vars already on each item.

Reply:
Just 8 bytes, and guns are not a performance critical item. Don't get sissy on vars. You should be getting sissy over MOB and TURF vars, MOB ones have real bloat.

Reply:
> guns are not a performance critical item

![image](https://user-images.githubusercontent.com/65828539/162742303-441e9610-44a0-4142-bc97-2b8392aa4fff.png)